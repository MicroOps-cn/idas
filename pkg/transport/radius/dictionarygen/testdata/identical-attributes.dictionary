ATTRIBUTE First-Name 200 string
ATTRIBUTE First-Name 200 string
