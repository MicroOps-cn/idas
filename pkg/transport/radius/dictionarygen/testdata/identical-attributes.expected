Constants:
	const (
		FirstName_Type radius.Type = 200
	)
Functions:
		func FirstName_Add(p *radius.Packet, value []byte) (err error)
		func FirstName_AddString(p *radius.Packet, value string) (err error)
		func FirstName_Del(p *radius.Packet)
		func FirstName_Get(p *radius.Packet) (value []byte)
		func FirstName_GetString(p *radius.Packet) (value string)
		func FirstName_GetStrings(p *radius.Packet) (values []string, err error)
		func FirstName_Gets(p *radius.Packet) (values [][]byte, err error)
		func FirstName_Lookup(p *radius.Packet) (value []byte, err error)
		func FirstName_LookupString(p *radius.Packet) (value string, err error)
		func FirstName_Set(p *radius.Packet, value []byte) (err error)
		func FirstName_SetString(p *radius.Packet, value string) (err error)
